### DISCUSSION TITLE
Noob question: No search results OOTB with Solr

### DISCUSSION THREAD

**Discussion by lolaslade** [schranz-search/schranz-search#407] [General]
Thanks so much for this library. I am definitely going to replace some of the adhoc code I had for pushing our entities to Solr even if I have to customise an adapter heavily.

However, OOTB using the getting started setup I am not getting any results from the Solr queries. Documents are indexed but in the Solr UI I am also only seeing three fields when I retrieve results: id, source and _version. I am getting results if I switch the engine to ElasticSearch so I think  I have things mostly correct.

**Comment by alexander-schranz** [comment]
@lolaslade can you share a little reproducer repository based on a failing test case or based on the .examples repository which show all your code and configuration you use.
